config/src/vespa/config/frt/frtconfigrequestfactory.h

```diff
@@ -23,6 +23,9 @@ class FRTConfigRequestFactory
 
     FRTConfigRequest::UP createConfigRequest(const ConfigKey & key, Connection * connection, const ConfigState & state, int64_t serverTimeout) const;
 private:
+#ifdef __clang__
+    [[maybe_unused]]
+#endif
     const int _protocolVersion;
```

Comment:
What is maybe unused ?

Reply:
It is an attribute that tells the compiler to not warn even if the variable appears to be unused.
It cannot be used when compiling with gcc, since that triggers another warning (about ignored attribute).

Reply:
If it is unused, just remove it.
We should avoid #ifdefs and other preprocessing mumbo jumbo causing different code for different compilers unless strictly necessary.